### ISSUE TITLE
Animated.Loop prevents VirtualizedList updates

### ISSUE THREAD

**Issue by willdawsonme** [facebook/react-native#16092]
### Is this a bug report?

Yes

### Have you read the [Contributing Guidelines](https://facebook.github.io/react-native/docs/contributing.html)?

Yes

### Environment

Environment:
  OS:  macOS Sierra 10.12.6
  Node:  6.9.1
  Yarn:  0.17.9
  npm:  3.10.10
  Watchman:  4.7.0
  Xcode:  Xcode 8.3 Build version 8E162
  Android Studio:  2.2 AI-145.3537739

Packages: (wanted => installed)
  react: 16.0.0-alpha.12 => 16.0.0-alpha.12
  react-native: 0.48.3 => 0.48.3

### Steps to Reproduce

1. Create any VirtualizedList and add a getItemLayout handler.
2. Start an Animated.Loop animated.
3. Scroll the list.

### Expected Behavior

I would expect the new cells to render.

### Actual Behavior

New cells do not render, only the cells that rendered initially are visible.

### Reproducible Demo

https://snack.expo.io/H18xSNvi-

# Comments

I believe part of the issue lies in `_scheduleCellsToRenderUpdate`: https://github.com/facebook/react-native/blob/3649fce12916f81a55be953dd14946d6101e71fa/Libraries/Lists/VirtualizedList.js#L1090-L1099.
 When scrolling, cell updates are scheduled and never called. Adding `useNativeDriver: true` to the child animation doesn't seem to help.

Removing the `getItemLayout` handler causes the list to calculate an `averageCellLength`. Now, the cell updates aren't scheduled but rendered immediately. Perhaps if a getItemLayout handler is provided an averageCellLength could be calculated which would let hiPri updates be rendered immediately.

Thanks

**Comment by rawrmaan** [comment]
I've got the same issue on 0.47.1. I noticed the same thing as you--removing `getItemLayout` fixes it, but that's absolutely not something I'd want to do. Glad you found the actual source code of the issue.

**Comment by janicduplessis** [comment]
VirtualizedList is probably using InteractionManager to schedule work but get's blocked by your looping animation. You can add `isInteraction: false` to your animation config and it should fix the issue.

**Comment by janicduplessis** [comment]
Maybe we could set isInteraction to false automatically when using Animated.loop since the main use case is for long running animations and it should not block InteractionManager.

**Comment by willdawsonme** [comment]
Ah perfect! Yeah that sounds like a sensible change, but at the very least a documentation update would be really helpful. Thank you!

What are your thoughts on the potential VirtualizedList `averageCellLength` improvement I mentioned at the end of the original issue? Happy to open up another ticket if you think it's worthwhile.

**Comment by rawrmaan** [comment]
@willdawsonme I submitted a doc update in #16316.
